base/metrics/persistent_memory_allocator.cc

```diff
@@ -1034,9 +1043,11 @@ void LocalPersistentMemoryAllocator::DeallocateLocalMemory(void* memory,
 #else
 #error This architecture is not (yet) supported.
 #endif
+#endif  // !defined(STARBOARD)
```

Comment:
I would recommend doing `#elif BUILDFLAG(IS_STARBOARD)` here between lines 1042-1043, it fits a little better

Reply:
You are right. I think it now works. I was able to revert the change here, and everything works fine.